packages/machine-extractor/examples/transition-description.machine.ts

```diff
@@ -0,0 +1,23 @@
+import { createMachine } from "xstate";
+
+export const transitionDescription = createMachine({
+	id: "TransitionDescriptionMachine",
+	type: "parallel",
+	states: {
+		Ticket: {
+			initial: "Open",
+			states: {
+				Open: {
+					on: {
+						cancel: {
+							actions: "cancel ticket",
+							target: "Canceled",
+							description: "This is description example",
+						},
+					},
+				},
+				Canceled: {},
+			},
+		},
+	},
+});
```

Comment:
@mattpocock hi, i've added the simple machine for example to show description inside transition, is this enough?